nav2_graceful_controller/src/parameter_handler.cpp

```diff
@@ -52,21 +54,26 @@ ParameterHandler::ParameterHandler(
     node, plugin_name_ + ".v_linear_max", rclcpp::ParameterValue(0.5));
   declare_parameter_if_not_declared(
     node, plugin_name_ + ".v_angular_max", rclcpp::ParameterValue(1.0));
+  declare_parameter_if_not_declared(
+    node, plugin_name_ + ".v_angular_min_in_place", rclcpp::ParameterValue(0.25));
   declare_parameter_if_not_declared(
     node, plugin_name_ + ".slowdown_radius", rclcpp::ParameterValue(1.5));
   declare_parameter_if_not_declared(
     node, plugin_name_ + ".initial_rotation", rclcpp::ParameterValue(true));
   declare_parameter_if_not_declared(
-    node, plugin_name_ + ".initial_rotation_min_angle", rclcpp::ParameterValue(0.75));
+    node, plugin_name_ + ".initial_rotation_tolerance", rclcpp::ParameterValue(0.25));
```

Comment:
That's pretty tight - we'd stop whenever the angle is > 14 deg off. I don't think that's a sensible default behavior to have the robot stop that much. Is that what's causing all the halt and rotates in the video that @ajtudela posted? 

Maybe the value before of ~43 deg was too high? But I think this is definitely too low

Reply:
An initial rotation only happens when a new plan arrives - I suppose if you have continuous replanning running, this would be a problem - but is that the default out of the box?

Reply:
> I suppose if you have continuous replanning running, this would be a problem - but is that the default out of the box?

Yes :-)

Reply:
I'll revert this change then, and add some notes to the documentation